packages/citrix_adc/data_stream/log/elasticsearch/ingest_pipeline/native.yml

```
@@ -337,7 +337,7 @@ processors:
       field:
         - citrix_adc.log.action
         - citrix_adc.log.browser
-        - citrix_adc.log.bytes.received
+        - citrix_adc.log.bytes.recieved
```

```suggestion
        - citrix_adc.log.bytes.received
```

This was misspelled in the fields definitions, so unfortunately, the misspelling must remain.